obal/data/roles/build_package/defaults/main.yaml

```diff
@@ -8,3 +8,4 @@ build_package_tito_releaser_args: []
 build_package_wait: true
 build_package_download_logs: false
 build_package_download_rpms: false
+build_package_waitrepo: false
```

Comment:
I would honestly love if this were true by default.

Reply:
We can do that, the only dependency is build_wait, and that's already true by default.